frontend/package.json

```diff
@@ -82,6 +83,7 @@
     "query-string": "^5.0.1",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-faux-dom": "^4.1.0",
```

Comment:
I think this is not used, and can be removed, right?

Reply:
si thly